packages/react-jss/src/createHoc.js

```diff
@@ -176,6 +209,7 @@ export default (stylesOrCreator, InnerComponent, options = {}) => {
         })
       }
 
+      // $FlowFixMe InnerComponent can be class or stateless, the latter doesn't have a defaultProps property
```

Comment:
it may have defaultProps property same way as class based

Reply:
yeah but flow doesn't have them https://github.com/facebook/flow/blob/master/lib/react.js#L99

Reply:
weird, we should look for an issue there for explanation or raise one .... really weird